maui-toc.html

```diff
@@ -182,6 +182,13 @@
             <li><a href="/maui/circular-charts/tooltip">Tooltip</a></li>
         </ul>
     </li>
+    <li>
```

Comment:
remove and add with existing SfDataForm list

Reply:
removed and added with existing SfDataForm list